R/ui.R

```diff
@@ -153,6 +149,15 @@ HarmonyMatrix <- function(
     }
     
     # determine if lambda is appropriate
+    if (!("lambda" %in% names(opt_args))){
+        lambda <- c(0.1, 10)
+    } else{
```

Comment:
Since we allow only range values of lambda you need to check whether lambda has length 2 before issuing this message.

Reply:
You mean to do all the lambda integrity checking before the warning message?